va/va.h

```diff
@@ -4224,8 +4232,14 @@ typedef struct _VASurfaceDecodeMBErrors {
 /**
  * After the application gets VA_STATUS_ERROR_DECODING_ERROR after calling vaSyncSurface(),
  * it can call vaQuerySurfaceError to find out further details on the particular error.
- * VA_STATUS_ERROR_DECODING_ERROR should be passed in as "error_status",
- * upon the return, error_info will point to an array of _VASurfaceDecodeMBErrors structure,
+ * VA_STATUS_ERROR_DECODING_ERROR should be passed in as "error_status".
+ *
+ * After the applications get VA_STATUS_HW_BUSY or VA_STATUS_SUCCESSFULL from vaSyncSurface(),
+ * it still can call vaQuerySurfaceError to find out further details to know if has real hw reset
+ * happened on this surface since umd and kmd could recover the context from reset with success in sometimes.
+ * VA_STATUS_ERROR_RESET should be passed in as "error_status".
+ *
+ * Upon the return, error_info will point to an array of _VASurfaceDecodeMBErrors structure,
```

Comment:
not clear, VA_STATUS_ERROR_RESET maybe return from vaSyncSurfaces, then use as error_status for this function?

Reply:
vaSyncSurface never returns VA_STATUS_ERROR_RESET. Should clear here, query with VA_STATUS_ERROR_RESET is not necessary, if application wants to know if the surface has error with hw reset, it could query it.

Reply:
`VA_STATUS_ERROR_RESET should be passed in as "error_status".` will change to:
`VA_STATUS_HW_BUSY or VA_STATUS_SUCCESSFULL should be passed in as "error_status".`

`VA_STATUS_HW_BUSY` is for legacy platform (TGL). `VA_STATUS_SUCCESSFULL` is for DG2 or the further platform.